src/components/idleUsers/card/index.tsx

```diff
@@ -17,20 +16,18 @@ const Card: FC<Props> = ({ user }) => {
   const profileUrl = `${MEMBERS_URL}/${user.username}`
 
   return (
-    <Link href={profileUrl}>
-      <a data-testid='profile-card' className={styles.card}>
+      <a data-testid='profile-card' className={styles.card} href={profileUrl} > 
         <Image
           src={userImg || DUMMY_PROFILE}
           alt={user.full_name}
           width={150}
           height={150}
           data-testid='user-image'
+          priority={true}
```

Comment:
won't setting a priority make our page load slow(er)?

Reply:
yeah also would like to know why are we setting this to true, as far as I read on the docs `You should use the priority property on any image detected as the [Largest Contentful Paint (LCP)](https://nextjs.org/learn/seo/web-performance/lcp) element`. So are these images largest element on the page. If yes then we can go with true else we can default to false.